CesiumGS/cesium
# Add extensibility to various classes

# Description

Add new functions to register various updaters and visualizers to support custom entities and data constructs.







## Issue number and link

No issue





## Testing plan



- Run the specs and make sure they all still pass
- Try the sandcastles related to entities, primitives and CZML and make sure they all still behave as intended.

# Author checklist
